build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
 
 	testCompile gradleTestKit()
 	testCompile 'junit:junit:4+'
+	testCompile 'com.vladsch.flexmark:flexmark-ext-tables:0.34.52'
```

Comment:
Since there are multiple flexmark dependencies, can you may create a variable for the version because i think its necessary to take for all the same version.